libs/core/src/components/pds-select/test/pds-select.spec.tsx

```diff
@@ -0,0 +1,229 @@
+      html: `<pds-select options='[{"value": "paul", "label": "Paul McCartney"}, {"value": "john", "label": "John Lennon"}]' />`,
+    });
+
+    const pdsSelect = page.root;
+    if (!pdsSelect || !pdsSelect.shadowRoot) {
+      throw new Error('pdsSelect or pdsSelect.shadowRoot is not available');
+    }
+    const select = pdsSelect.shadowRoot.querySelector('select');
+
+    // Change value to 'paul'
+    if (select) {
+      select.value = 'paul';
+      select.dispatchEvent(new Event('change'));
+      await page.waitForChanges();
+
+      // Check if the value is updated
+      expect(select.value).toBe('paul');
+
+      // Change value to 'john'
+      select.value = 'john';
+      select.dispatchEvent(new Event('change'));
+      await page.waitForChanges();
+
+      // Check if the value is updated
+      expect(select.value).toBe('john');
+
+      // Change value to empty
+      select.value = '';
+      select.dispatchEvent(new Event('change'));
+      await page.waitForChanges();
+
+      // Check if the value is updated
+      expect(select.value).toBe('');
+    } else {
+      throw new Error('select element is not available');
+    }
+  });
+
+  it('emits pdsSelect event on value change', async () => {
+    const page = await newSpecPage({
+      components: [PdsSelect],
+      html: `<pds-select options='[{"value": "paul", "label": "Paul McCartney"}, {"value": "john", "label": "John Lennon"}]' />`,
+    });
+
+    const pdsSelect = page.root;
+    if (!pdsSelect || !pdsSelect.shadowRoot) {
+      throw new Error('pdsSelect or pdsSelect.shadowRoot is not available');
+    }
+    const select = pdsSelect.shadowRoot.querySelector('select');
+
+    const pdsSelectEvent = jest.fn();
+    page.win.addEventListener('pdsSelect', pdsSelectEvent);
+
+    // Change value to 'paul'
+    if (select) {
+      select.value = 'paul';
+      select.dispatchEvent(new Event('change'));
+      await page.waitForChanges();
+
+      // Check if the event is emitted
+      expect(pdsSelectEvent).toHaveBeenCalled();
+    } else {
+      throw new Error('select element is not available');
+    }
+  });
+
+  it('renders autocomplete attribute when property is passed', async () => {
+    const page = await newSpecPage({
+      components: [PdsSelect],
+      html: `<pds-select autocomplete="off" component-id="field-1" label="Name"></pds-select>`,
+    });
+
+    expect(page.root).toEqualHtml(`
+      <pds-select autocomplete="off" component-id="field-1" label="Name">
+        <mock:shadow-root>
+          <div class="pds-select">
+            <label htmlFor="field-1">Name</label>
+            <select autocomplete="off" class="pds-select__field" id="field-1"></select>
+            <div aria-hidden="true" class="hidden">
+              <slot></slot>
+            </div>
+            <pds-icon class="pds-select__select-icon" icon="${enlarge}"></pds-icon>
+          </div>
+        </mock:shadow-root>
+      </pds-select>
+    `);
+  });
+});
```

Comment:
Overall good job on the `.spec` test. These are meant to only test a piece of functionality (functions) and you did that.